config/prod/JKG-ec2-v5.yaml

```diff
@@ -0,0 +1,29 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v5.yaml
+stack_name: JKG-SB-test-ec2
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: "jake.gockley@sagebase.org"
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.xlarge"
+  # (Optional) ID of the base AMI (supported distros: AWS linux)
+  #AMIId: "ami-0d051218cbaf3405a"      # AMI with my R packages
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  #VolumeSize: "50"
```

Comment:
Did you want an instance larger than 8GB?

Reply:
Ideally I'd like to keep it small. Can I mount an s3 larger than 8GB, or is 8GB always the limit?

Reply:
This is referring to the volume on the EC2, not S3.  Look at the comment on the previous line for default and limit: "# (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB"

Reply:
This is referring to the volume on the EC2, not S3. Look at the comment on the previous line for default and limit: "# (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB"